pkg/controllers/awsloadbalancercontroller/credentials_request.go

```diff
@@ -78,6 +78,19 @@ func (r *AWSLoadBalancerControllerReconciler) ensureCredentialsRequest(ctx conte
 	return current, nil
 }
 
+func (r *AWSLoadBalancerControllerReconciler) credentialsSecretProvisioned(ctx context.Context, cr *cco.CredentialsRequest) (bool, error) {
```

Comment:
This function should return a `false, nil` when the error is `IsNotFound` so that the caller doesn't have to handle it.